include/cantera/thermo/IdealGasPhase.h

```diff
@@ -401,6 +401,10 @@ class IdealGasPhase: public ThermoPhase
         return 1.0 / temperature();
     }
 
+    virtual double soundSpeed() const {
+        return sqrt(cp_mole() / cv_mole() * GasConstant / meanMolecularWeight() * temperature());
+    }
```

Comment:
A small nitpick here -- can you please format this so it fits within our line length limit of 88 characters? It might be better to move the implementation into the `.cpp` file as well.

Reply:
@speth Should be good now. I also squashed some commits.